sources/packages/web/src/components/common/Reports.vue

```diff
@@ -62,8 +63,8 @@ export default defineComponent({
       );
     };
     const formChanged = async (form: FormIOForm, event: FormIOChangeEvent) => {
-      // Populates the program year select component if required.
-      if (event.changed.component.key === REPORT_TYPE_DROPDOWN_KEY) {
+      // Populates the program year select component, institution select component if required.
+      if (event.changed?.component.key === REPORT_TYPE_DROPDOWN_KEY) {
```

Comment:
`event.changed` is not marked as optional in the interface.
We should change the interface or remove the null check ("?").